### ISSUE TITLE
Emulator crashes when opening any patient file

### ISSUE THREAD

**Issue by viniciusboson** [projectbuendia/client#7]
The development repository seems to have an issue.

I can't see the stack trace from #2, but i believe it's a different issue.

I cloned the project and  followed the instructions on its README.

The error is happening either changing the changing the default OpenMRS server settings to local (10.0.2.2) or maintaining its default (dev.projectbuendia.org).

The stack trace for this error is:

``` java
09-14 20:32:33.626    3173-3632/org.projectbuendia.client.dev E/AndroidRuntime﹕ FATAL EXCEPTION: AsyncTask #2
    Process: org.projectbuendia.client.dev, PID: 3173
    java.lang.RuntimeException: An error occured while executing doInBackground()
            at android.os.AsyncTask$3.done(AsyncTask.java:300)
            at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:355)
            at java.util.concurrent.FutureTask.setException(FutureTask.java:222)
            at java.util.concurrent.FutureTask.run(FutureTask.java:242)
            at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:231)
            at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
            at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
            at java.lang.Thread.run(Thread.java:841)
     Caused by: java.lang.ClassCastException: java.lang.Object[] cannot be cast to java.lang.Void[]
            at org.projectbuendia.client.models.AppModel$FetchTypedCursorAsyncTask.doInBackground(AppModel.java:268)
            at android.os.AsyncTask$2.call(AsyncTask.java:288)
            at java.util.concurrent.FutureTask.run(FutureTask.java:237)
            at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:231)
            at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
            at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
            at java.lang.Thread.run(Thread.java:841)
```

**Comment by viniciusboson** [comment]
I've started from the scratch again, setting up all my the environment and i can't reproduce the error no more.
I'm closing this issue. If this happens with someone else, please, reopen it.

**Comment by viniciusboson** [comment]
This issue happens when one try to compile the client with a java 8 version. Fixed on PR #96
